sf/draft procurement/stmt of qual.md

```diff
@@ -31,22 +31,26 @@ Play 2 – Address the Whole Experience from Start to Finish
   
 Play 3 – Make it Simple and Intuitive
   * The products or services for our customers/clients do not require knowledge of the organizational structure or our agency to complete the process.
-  * We coordinated with our customers/clients to make design and content decisions based on user research, usability testing, and analytics.
-  * For all user-facing features built over the last quarter, we coordinated with our customers/clients to use plain language that is familiar to the user and easy to understand.
+  * We coordinated with our customers/clients to make design and content decisions based on accessibility needs, user research, usability testing, and analytics.
+  * For all user-facing features built recently, we coordinated with our customers/clients to use plain language that is familiar to the user, easy to understand, and possible to translate across cultures and languages.
```

Comment:
- Added the need for translatable content given that Smithsonian museums are some of the most highly trafficked museums in the world. This fact means that vast numbers of visitors will have a language other than english as their primary language.